src/modules/auth/LogIn.page.tsx

```diff
@@ -1,7 +1,22 @@
 import useLazyLoadResourceHook from '@shared/hooks/lazyLoadResource.hook';
 import { LogIn as LogInComponent } from './components/LogIn/LogIn.component';
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { RootState } from '@store/index.store.ts';
 
 export default function LogIn(): JSX.Element {
+  const navigate = useNavigate();
+  const { token, isTrainer } = useSelector((state: RootState) => state.auth);
+  useEffect(() => {
```

Comment:
My preference would be to add a new line before `useEffect`, so that not everything feels so cramped up.